push-build.sh

```diff
@@ -122,10 +122,10 @@ RELEASE_BUCKET=${FLAGS_bucket:-"kubernetes-release-dev"}
 # This will canonicalize the path
 KUBE_ROOT=$(pwd -P)
 
+# TODO: this should really just read the version from the release tarball always
 USE_BAZEL=false
 if release::was_built_with_bazel $KUBE_ROOT $FLAGS_release_kind; then
   USE_BAZEL=true
-  bazel build //:version
```

Comment:
`//:version` is a dependency of the //build:release-tars afaict anyhow, invoking it here is pointless and slow.

Reply:
We used to do this to verify that the build was actually up-to-date:
https://github.com/kubernetes/release/blob/8a8da81afc65a8d191b497b733455e8fcb43204f/push-build.sh#L126-L130

This has since been lost to time.

Reply:
well, apparently it was lost in https://github.com/kubernetes/release/pull/444.